mkdocs.yml

```diff
@@ -38,12 +42,27 @@ theme:
     - navigation.tabs
     - toc.integrate
 
+# Custom styles
+extra_css:
+  - css/buttons.css
+
 #Enable macros
 plugins:
   - search
   - macros    
 
 extra:
+  downloads:
+    default: "https://github.com/xournalpp/xournalpp/releases"
+    nightly: "https://github.com/xournalpp/xournalpp/releases/tag/nightly"
+    windows:
+      stable: "https://github.com/xournalpp/xournalpp/releases/download/1.0.20/xournalpp-1.0.20-windows.zip"
+    macos:
+      stable: "https://github.com/xournalpp/xournalpp/releases/download/1.0.20/xournalpp-1.0.20-macos.zip"
+    linux:
+      flatpak: "https://flathub.org/apps/details/com.github.xournalpp.xournalpp"
+      appimage: "https://github.com/xournalpp/xournalpp/releases/download/1.0.20/xournalpp-1.0.20-x86_64.AppImage"
```

Comment:
There is an AppImage for the stable version and an AppImage for the nightly release. Both should have a button.